back-office/advanced-parameters/database/advanced_parameters-database-sql_manager.md

```diff
@@ -1,8 +1,59 @@
 # SPECIFICATIONS - ADVANCED PARAMETERS &gt; DATABASE &gt; SQL MANAGER
 
-## Grid
+The SQL manager is a complex feature, which should be reserved for technical people who know how to explore a database using the SQL language. In return for this complexity, it can be extremely powerful and proves immensely useful to those who master it.
 
-Default action in the button: Export Other available actions in the menu: View, Edit, Delete Row action: View
+## 1.7.8 Version 
+The page is going to be migrated into a module. The module will be downloadable for merchants who need it.
+The feature to hide the password or the secure key will be removed since we can't fully hide them since with an alias the keys or passwords can be displayed.
 
-Row action does not apply to the checkbox column.
+## 1.7.7.0 Version
+
+This tool enables you to perform SQL queries directly on the PrestaShop database, and save them for use at any later time. 
+
+For instance, using this tool and your knowledge of SQL, you could create a reusable query giving you an updated list of the clients who are subscribed to your newsletter or get a list of products in HTML or CSV format.
+
+For security reasons, some types of queries are not allowed: UPDATE, DELETE, CREATE TABLE, DROP, etc. In short, you can only read data (SELECT query).
+
+At first, the SQL Manager page displays 2 messages:
+- Alert info:
+
+  "How do I create a new SQL query?
+    - Click "Add New".
+    - Fill in the fields and click "Save".
+    - You can then view the query results by clicking on the Edit action in the dropdown menu
+    - You can also export the query results as a CSV file by clicking on the Export button
+"
+
+- Alert warning:
+
+"When saving the query, only the "SELECT" SQL statement is allowed."
+
+### SQL Manager List
+
+Below the messages, the page displays the SQL Manager list with the request's ID, SQL query name, and action columns.
+The request's actions allow us to download the request's result into a CSV file, view the request's result on the page, delete the request, and edit it.
+To summarize the actions cab be:
+- Export. Performs the query, and have it downloaded by you in CSV format.
```

Comment:
What do you mean by "performs the query" ? 
The query is just downloaded in CSV format

Reply:
Execute the query to retrieve the data and then download it. It is not the query that is exported but the results of the query. I have doubts now, I'm going to check

Reply:
Ah, yes my bad you are right it's the results of the query !
So the query is well executed to get the results :)